visual_behavior/visualization/extended_trials/daily.py

```diff
@@ -323,12 +328,19 @@ def make_daily_figure(
     make_rolling_response_probability_plot(hit_rate, fa_rate, ax[2], palette=palette)
     mean_rate = np.mean(check_responses(df_nonaborted, reward_window=reward_window) == 1.0)
     ax[2].axvline(mean_rate, color='0.5', linestyle=':')
-
-    peak_dprime = session_metrics.peak_dprime(extended_trials)
-    if not np.isnan(peak_dprime):
-        make_rolling_dprime_plot(d_prime, ax[3], peak_dprime=peak_dprime)
-    else:
-        make_rolling_dprime_plot(d_prime, ax[3])
+    
+    # CB added to display both the trial adjusted and legacy dprime metric on same plot
+    for apply_trial_number_limit, first_valid_trial, append_ticks, color in zip([False, True], [50, 100], [False, True], ['black', 'grey']):
+        hit_rate, fa_rate, d_prime = get_response_rates(
+                df_nonaborted,
+                sliding_window=sliding_window,
+                apply_trial_number_limit=apply_trial_number_limit
+                )
+        peak_dprime = session_metrics.peak_dprime(extended_trials, first_valid_trial=first_valid_trial, apply_trial_number_limit=apply_trial_number_limit)
+        if not np.isnan(peak_dprime):
+            make_rolling_dprime_plot(d_prime, ax[3], peak_dprime=peak_dprime, append_ticks=append_ticks, color=color)
+        else:
+            make_rolling_dprime_plot(d_prime, ax[3], append_ticks=append_ticks, color=color)
```

Comment:
Is it obvious which line belongs to which condition? It seems like a legend would be a good idea.

Reply:
Agreed. Done!

Reply:
@dougollerenshaw also, the linting errors that are being caught below aren't for any of the code I modified (or even in the same file). They are primarily here:

visual_behavior/visualization/qc/dash_app/app.py

I'll go ahead and fix them. Not totally sure how this happened...